src/components/FieldImage.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getTranslator } from '../reducers/translation';
+import styles from './FieldImage.scss';
+
+const fileInstance = new FileReader(); // eslint-disable-line no-undef
+
+const DropzoneImage = ({ i18n, input }) => (
+  <div>
+    <Dropzone
+      accept={'image/*'}
+      maxSize={300000}
+      multiple={false}
+      className={styles.overide}
+      name={input.name}
+      onDrop={(imageData) => {
+        fileInstance.readAsDataURL(imageData[0]);
+        fileInstance.onloadend = () => input.onChange(fileInstance.result);
+      }}
+    >
+      <div>
+        <p className={styles.header} >
+          <i className={`material-icons ${styles.logo}`}>{i18n('publish')}</i> &nbsp; &nbsp; {i18n('Click here to upload a file')} </p>
```

Comment:
`<i>` tag should not be used. Please read the notes in [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i)